lib/js-yaml/dumper.js

```diff
@@ -132,6 +132,30 @@ function State(options) {
 }
 
 
+function indentString(string, spaces) {
+  var ind = common.repeat(' ', spaces);
+  var position = 0;
+  var next = -1;
+  var result = '';
+  var line;
+  var length = string.length;
```

Comment:
Please use single `var` statement for the whole block of assignments. See any other place in our code for a reference.